example-local.yaml

```diff
@@ -4,8 +4,9 @@ substitutions:
 
 packages:  # Change this file to the correct one(s) for your device
   econet-hpwh: !include econet_heatpump_water_heater.yaml
-  # econet-hvac: !include econet_hvac.yaml
-  # econet-hvac-odu: !include econet_hvac_odu.yaml  # Must be included with econet_hvac.yaml
+  # econet-hvac: !include econet_hvac_furnace.yaml
+  # econet-hvac: !include econet_hvac_airhandler.yaml
```

Comment:
Yes. Although "airhandler" needs to be "air_handler"

Reply:
And you get the expected results (i.e. sum of the two files and not just one of them over-writing the other)? Interesting.

Reply:
Yep. Its essentially the same thing we do with everything and econet_base.yaml already.

Reply:
Oh,  I see. As long as you just use one of the two you're fine. It won't work if you use both of them (since you can't have a duplicate of the `econet-hvac` key) but that's not your intended usecase.

Reply:
I would still name them differently in case users want to include both (e.g. their furnace has air handling capabilities due to attached Heat Pump):
```yaml
# econet-hvac-furnace: !include econet_hvac_furnace.yaml
# econet-hvac:-air-handler: !include econet_hvac_airhandler.yaml
```